apps/tsilhqotin-language-hub/src/Components/Pages/Apps/AppsDetail.tsx

```diff
@@ -0,0 +1,101 @@
+import { Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material';
+
+export function AppsDetail() {
```

Comment:
Why is this called `AppsDetail`? It is clearly an `index` (`list`) scoped component. I would move `appData` to `Apps` (the page) and do the `appData.map` there. The `AppDetail` (singular not `AppsDetail`) should be a presentational component that takes in a single `appInfo` (one element of `appData`) as props and returns the TSX for the view of one single App's info.

Reply:
We fixed this.